third_party/terraform/tests/resource_bigtable_instance_iam_test.go

```diff
@@ -0,0 +1,211 @@
+			{
+				// Test IAM Binding creation
+				Config: testAccBigtableInstanceIamBinding_basic(instance, account, role),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"google_bigtable_instance_iam_binding.binding", "role", role),
+				),
+			},
+			{
+				ResourceName:      "google_bigtable_instance_iam_binding.binding",
+				ImportStateId:     importId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+			{
+				// Test IAM Binding update
+				Config: testAccBigtableInstanceIamBinding_update(instance, account, role),
+			},
+			{
+				ResourceName:      "google_bigtable_instance_iam_binding.binding",
+				ImportStateId:     importId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccBigtableInstanceIamMember(t *testing.T) {
+	t.Parallel()
+
+	instance := "tf-bigtable-iam-" + acctest.RandString(10)
+	account := "tf-bigtable-iam-" + acctest.RandString(10)
+	role := "roles/bigtable.user"
+
+	importId := fmt.Sprintf("projects/%s/instances/%s %s serviceAccount:%s",
+		getTestProjectFromEnv(),
+		instance,
+		role,
+		serviceAccountCanonicalEmail(account))
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				// Test IAM Binding creation
+				Config: testAccBigtableInstanceIamMember(instance, account, role),
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"google_bigtable_instance_iam_member.member", "role", role),
+					resource.TestCheckResourceAttr(
+						"google_bigtable_instance_iam_member.member", "member", "serviceAccount:"+serviceAccountCanonicalEmail(account)),
+				),
+			},
+			{
+				ResourceName:      "google_bigtable_instance_iam_member.member",
+				ImportStateId:     importId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccBigtableInstanceIamPolicy(t *testing.T) {
+	t.Parallel()
+
+	instance := "tf-bigtable-iam-" + acctest.RandString(10)
+	account := "tf-bigtable-iam-" + acctest.RandString(10)
+	role := "roles/bigtable.user"
+
+	importId := fmt.Sprintf("projects/%s/instances/%s",
+		getTestProjectFromEnv(), instance)
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				// Test IAM Binding creation
+				Config: testAccBigtableInstanceIamPolicy(instance, account, role),
+			},
+			{
+				ResourceName:      "google_bigtable_instance_iam_policy.policy",
+				ImportStateId:     importId,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func testAccBigtableInstanceIamBinding_basic(instance, account, role string) string {
+	return fmt.Sprintf(testBigtableInstanceIam+`
+
+resource "google_service_account" "test-account1" {
+  account_id   = "%s-1"
+  display_name = "Dataproc IAM Testing Account"
```

Comment:
This can be done in a followup so we don't end up with annoying merge stuff, but can these be changed so they don't say Dataproc?